src/services/message/index.js

```diff
@@ -15,14 +15,89 @@ class Service {
 
   constructor(options) {
     this.options = options || {};
-    this.docs = docs;
-  }
 
-  get(id,params) {
-      console.log('[INFO] get message ' + id);
-      return Message.findOne({_id:id});
+    this.docs = {
```

Comment:
Please use the extra file for the api documentation. I think while merging this was added again.